### PR TITLE
Add comment highlighting the log level that ConsoleEventListener writes to

### PR DESCRIPTION
**Pull request by abhipsaMisra**

### CODE REVIEW

**Review by drwill-ms — APPROVED** [review]

File: iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs [hunk 1]
```diff
@@ -47,9 +47,10 @@ public static async Task Main(string[] args)
                 });
             s_logger = loggerFactory.CreateLogger<Program>();
 
+            // Instantiating this seems to do all we need for outputting SDK events to our console log.
```
Comment (jamdavi) [inline iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs:50]: Should we possibly add a comment at line 46 above?  For example:

```csharp
new ColorConsoleLoggerConfiguration
{
    MinLogLevel = LogLevel.Debug,
    // Uncomment the line below to get ALL logs 
    // MinLogLevel = LogLevel.Trace, 
});

or

```csharp
new ColorConsoleLoggerConfiguration
{
    MinLogLevel = LogLevel.Debug, // Set this to LogLevel.Trace to get ALL logs 
});
```

File: iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs [hunk 2]
```diff
@@ -47,9 +47,10 @@ public static async Task Main(string[] args)
...
+            // The SDK logs are written at Trace level. To view SDK logs change the MinLogLevel to Trace.
             const string SdkEventProviderPrefix = "Microsoft-Azure-";
-            // Instantiating this seems to do all we need for outputting SDK events to our console log
-            _ = new ConsoleEventListener(SdkEventProviderPrefix, s_logger);
+            _ = new ConsoleEventListener(SdkEventProviderPrefix, logger);
```
Comment (brycewang-microsoft) [inline iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs:53]: It seems that we didn't declare or use `logger` in the TemperatureController sample here, should we align this to `s_logger` instead?

File: iot-hub/Samples/device/ConventionBasedOperations/Thermostat/Program.cs [hunk 3]
```diff
@@ -43,9 +43,10 @@ public static async Task Main(string[] args)
                 });
             s_logger = loggerFactory.CreateLogger<Program>();
 
+            // Instantiating this seems to do all we need for outputting SDK events to our console log.
+            // The SDK logs are written at Trace level. To view SDK logs change the MinLogLevel to Trace.
             const string SdkEventProviderPrefix = "Microsoft-Azure-";
-            // Instantiating this seems to do all we need for outputting SDK events to our console log
-            _ = new ConsoleEventListener(SdkEventProviderPrefix, s_logger);
+            _ = new ConsoleEventListener(SdkEventProviderPrefix, logger);
```
Comment (brycewang-microsoft) [inline iot-hub/Samples/device/ConventionBasedOperations/Thermostat/Program.cs:49]: Same `logger` issue here?

File: iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs (hunk 1 above)
Comment (abhipsaMisra) [reply iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs:50]: yeah, I like that

File: iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs (hunk 2 above)
Comment (abhipsaMisra) [reply iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs:53]: nice catch, copy-paste mistake!

**Pull request merged**